.jshintrc

```diff
@@ -1,6 +1,4 @@
 {
-  "camelcase": true,
-  "curly": true,
```

Comment:
Here's a link to the most updated style guides we use for the API - can we get them implemenetd here?

https://gist.github.com/pon/5047ec308e4475528a19

Thanks!